### PR TITLE
profiles: SelectUniqueTokenSheet android tweaks

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-4228
Figma link (if any):

## What changed (plus any additional context for devs)
use reg stack for `SelectUniqueTokenSheet` + behavior tweaks so scrolling, etc works well on android

## What to test

https://user-images.githubusercontent.com/29204161/183522775-49dfb79a-0ee0-468a-9641-737e90193a7f.mov

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]
https://www.loom.com/share/0e5334e493dd451b903d9203731bbc96 🚀

**Review by estebanmino — APPROVED** [review]
PoDQA 2 https://www.loom.com/share/fd1dcce326fa4025a8db852264a24570 with scrolling

**Review by jxom — APPROVED** [review]

File: src/navigation/effects.js [hunk 1]
```diff
@@ -443,6 +443,18 @@ export const sheetPreset = ({ route }) => {
...
 };
+export const selectUniquePreset = () => {
+  return {
+    cardOverlayEnabled: true,
+    cardShadowEnabled: true,
+    cardStyle: { backgroundColor: 'transparent' },
+    cardStyleInterpolator: sheetStyleInterpolator(0.7),
+    cardTransparent: true,
+    gestureDirection: 'vertical',
+    gestureResponseDistance: gestureResponseDistanceFactory(300),
```
Comment (osdnk) [inline src/navigation/effects.js:454]: what

**Comment by osdnk** [comment]
this code doesn't make much of sense and the original issue was already resolved by #3955

**Pull request merged**